dropwizard-testing/src/test/java/io/dropwizard/testing/junit5/TestEntity.java

```diff
@@ -34,7 +31,7 @@ long getId() {
     }
 
     String getDescription() {
-        return description;
+        return requireNonNull(description);
```

Comment:
What is that for?

Throwing a `NullPointerException` in a *getter* is very bad practice.

Reply:
Actually this one comes from this class: https://github.com/dropwizard/dropwizard/blob/master/dropwizard-testing/src/test/java/io/dropwizard/testing/junit/TestEntity.java